.cirrus.yml

```diff
@@ -106,8 +115,9 @@ ws_scan_task:
     folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   whitesource_script:
     - source cirrus-env QA
-    - source set_maven_build_version $BUILD_NUMBER
-    - mvn clean install -DskipTests -DskipTypeshed -Denforcer.skip=true -Dskip.its=true
+#   TODO: MS This can be more optimized, let's identify what exact task needs to be run
+    - ./gradlew --no-daemon --info --stacktrace --console plain --no-daemon build -x test -x runTypeshed
+    - source ./export_ws_variables_from_gradle
     - source ws_scan.sh
```

Comment:
I think there is a workaround in `wss-unified-agent.config` to ask mend to also analyze the .m2 folder:
`maven.m2RepositoryPath=.m2/repository`
This workaround is not ideal and generates a lot of false positives in Mend.
But it is here because Mend was not able to find dependencies otherwise.
And now with the gradle build, I think the `.m2/repository` workaround does not work anymore with Mend.